app/views/admin/poll/shifts/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_for @shift, as: :shift, url: admin_booth_shifts_path do |f| %>
+  <%= render "shared/errors", resource: @shift %>
+
+  <fieldset class="fieldset">
+    <legend>
+    	<%= t("admin.poll_shifts.new.new_shift") %>
+    </legend>
+    
+    <div class="small-12 medium-4 column">
+      Officer: <%= @officer.name %>
```

Comment:
Maybe `Officer` should be a translated string?

Reply:
Done! 👌